src/scripting/Elsa.Scripting.Liquid/Services/LiquidHandler.cs

```diff
@@ -33,7 +33,7 @@ public LiquidHandler(ILiquidTemplateManager liquidTemplateManager, IMediator med
 
         private async Task<TemplateContext> CreateTemplateContextAsync(ActivityExecutionContext activityExecutionContext, CancellationToken cancellationToken)
         {
-            var context = new TemplateContext();
+            var context = new TemplateContext(new TemplateOptions());
```

Comment:
This was causing test failures when the Liquid tests were run all together, my *guess* is that by default, Fluid mutates some static options object. So one test (the one asserting that config access can be enabled) adds the `Configuration` keyword and it persists into the next test causing it to fail. This ensures that getting a unique options object (and thus a unique member access strategy) for every test run. I don't think it would have any impact on consumers though

Reply:
Interesting. I agree, looks harmless enough.